Resources/doc/index.markdown

```diff
@@ -4,7 +4,7 @@
 
 ``` yaml
 "require": {
-    "lunetics/locale-bundle": "2.2.*",
+    "lunetics/locale-bundle": "2.4.*",
```

Comment:
i would not do that, as its better to limit to one major version. when
the version bumps to 3.0 it means there are BC breaks and you don't want
to handle them when doing "composer update"